src/shared/locales/all.js

```diff
@@ -1,9 +1,11 @@
 import eleLocaleEn from 'element-ui/lib/locale/lang/en'
+import eleLocalePtBr from 'element-ui/lib/locale/lang/pt-br'
```

Comment:
Please keep the variable naming case uniform.
`eleLocalePtBr` => `eleLocalePtBR`

Reply:
Sorry. Fixed!